assets/dev/js/editor/controls/media.js

```diff
@@ -50,10 +53,12 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 
 		if ( [ 'image', 'svg' ].includes( mediaType ) ) {
 			this.ui.mediaImage.css( 'background-image', url ? 'url(' + url + ')' : '' );
-
```

Comment:
revert the lie removal (to have less diff)

Reply:
Done